test/AutoRest.TestServer.Tests/Mgmt/OutputLibrary/OutputLibraryTestBase.cs

```diff
@@ -22,6 +22,7 @@
 
 namespace AutoRest.TestServer.Tests.Mgmt.OutputLibrary
```

Comment:
I would like to see more tests on CSharpProjConfiguration

Reply:
added